src/components/AvatarCropModal/AvatarCropModal.js

```diff
@@ -334,8 +340,11 @@ const AvatarCropModal = (props) => {
             .then((newImage) => {
                 props.onClose();
                 props.onSave(newImage);
+            })
+            .catch(() => {
+                isLoading.value = false;
             });
-    }, [originalImageHeight.value, originalImageWidth.value, scale.value, translateX.value, imageContainerSize, translateY.value, props, rotation.value]);
+    }, [originalImageHeight.value, originalImageWidth.value, scale.value, translateX.value, imageContainerSize, translateY.value, props, rotation.value, isLoading]);
```

Comment:
Shouldn't this be `isLoading.value`?

Reply:
Both are fine but I think `isLoading` is better for optimization.
If we add `isLoading.value` as dependency of `useCallback`, `cropAndSaveImage` function is re-calculated whenever that value changes.
If we set value to `isLoading.value`, it's updated instantly (`useSharedValue`'s feature), even if we use `useCallback` without dependency. So it's safe to add just `isLoading` as a dependency.

Reply:
I noticed that `isLoading` should be added as a dependency because its value is changed inside `useCallback`.
If we don't add that dependency but just `isLoading.value`, lint fails:
<img width="924" alt="lint" src="https://user-images.githubusercontent.com/108292595/231743799-060acf41-b752-4cbe-b37f-de4d7bf2c0a4.png">

Reply:
Ok, thanks for confirming that. 👍